.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
```

Comment:
The line above should be updated to `run: mkdir -p build && cd build && CC=$(command -v clang-10) CXX=$(command -v clang++-10) cmake -GNinja .. && ninja`